admin/App.js

```diff
@@ -156,7 +163,9 @@ export class App {
             },
             {
                 id: 'preferences',
-                switcherPosition: 'left',
+                componentProps: {
```

Comment:
I see why we need this now -- am going to look a bit more carefully -- can we specify the children as an array of functions that would produce elements and  let us be more general?

I am looking at this today and will report back

Reply:
Would love to discuss this some more.  Realizing that the abstraction here of child components-to-be into configs for the model (vs. declared children) might be one of the more surprising aspects of the overall component. 

I wonder if we could still achieve lazy-instantiation while keeping the parent / children relationships defined within the component tree - would that be more standard?